Contributing.md

```diff
@@ -128,7 +128,7 @@ The following branches should always be there. Do not fork them directly, always
 
 ### Releasing workflow
 
-We follow [semver](http://semver.org/). We're in `0.x` at the moment, however, as Stylus is already widely used, we don't introduce backwards-incompatible changes to our minor releases.
+We currently apply [semver](http://semver.org/). We're in `0.x` at the moment, however, as Stylus is already widely used, we don't introduce backwards-incompatible changes to our minor releases.
```

Comment:
what difference? The project is 0.x anyway too

Reply:
I think `apply` is more correct. The `current` means to emphasize the workflow state, it can be omitted. I still think it's useful though.

Reply:
@vendethiel Btw, you seems good add correcting grammar and spelling. Could you help me to check my other PR (in this repo too) https://github.com/stylus/stylus/pull/2496